src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -140,14 +130,15 @@ private Optional<User> getUserFromCookie(final Cookie customJWTCookie) {
 
     private Cookie getCustomJwtCookieFromRequest(final HttpServletRequest request) {
         final Cookie customJWTCookie = WebUtils.getCookie(request, userServiceCookieName);
-        if (customJWTCookie == null) throw new UnauthorizedException("No " + userServiceCookieName + " cookie found");
+        if (customJWTCookie == null)
+            throw new UnauthorizedException("No " + userServiceCookieName + " cookie found");
```

Comment:
nitpicking here but we can edit the language here to be more in line with convention

```suggestion
            throw new UnauthorizedException(userServiceCookieName + " cookie not found");
```